### ISSUE TITLE
Seek cursor doesn't correspond with playback position on iOS

### ISSUE THREAD

**Issue by atlesque** [sudara/alonetone#1172]
To reproduce:

- Open a playlist page on an iOS device (iPhone, iPad, ...)
- Play a track
- Seek forward in the track
- Seek backward again
- The playback cursor is misaligned by about 1-2 seconds

Expected: The cursor to keep correct alignment.

This bug is also present on the user's page, when playing a track and seeking, the cursor misses alignment by the same amount,

**Comment by sudara** [comment]
Thanks for all of the QA! I like your reproduction steps, nice work.

Question: do you mean this gray line to the left of the orange playback line? It stays in place while the orange playhead moves?

![IMG_1168](https://user-images.githubusercontent.com/472/152009881-30657b42-9147-4620-8f81-d0f975b4ae0b.jpg)

We should remove that completely on mobile, as it's an "on hover" cursor to show that the area is seekable.

But maybe you mean something else, like you expect the orange playhead to be further forward or backward based on some math/metric?...

**Comment by atlesque** [comment]
Yes, sorry for the late reply: I meant the orange playhead which seems off, probably due to some math which is different on mobile display.
